docs/reference/flavors/index.md

```diff
@@ -0,0 +1,50 @@
+# Flavors
+
+Any server instance running in {{extra.brand}} Cloud has a **flavor**,
+which defines the number of virtual CPU cores, the amount of virtual
+RAM, and other performance-related factors.
+
+
+## Naming convention
+
+Flavor names in {{extra.brand}} follow a convention, which can be
+summarized as `X.YcZgb`:
+
+* `X` stands for a lowercase letter identifying the [compute
+  tier](#compute-tiers), with `b` representing the general-purpose
+  tier. It is always followed by a full-stop (`.`).
+* `Y` stands for the number of virtual CPU cores. This number is
+  always followed by the letter `c`.
+* `Z` stands for the allocated amount of virtual RAM, in
+  [gibibytes](https://en.wikipedia.org/wiki/Gigabyte#Base_2_(binary)). This
+  number is always followed by the string `gb`.
+
+For example, the flavor named `b.4c32gb` would be used for a
+general-purpose compute instance with 4 cores and 32 GiB RAM.
+
+
+## Compute tiers
+
+{{extra.brand}} Cloud defines the following compute tiers:
+
+* `b`: General purpose. This is the default compute tier. Instances
+  launched with matching flavors use highly available network-attached
+  storage. This makes them flexible to migrate within the
+  {{extra.brand}} Cloud infrastructure, without interruption.
+* `s`: High-performance local storage. Instances
+  launched with matching flavors use local, directly-attached
+  storage. This generally provides higher throughput and lower
+  latency for I/O intensive applications, but instances launched with
+  these flavors must configure their own high availability and data
+  replication. 
```

Comment:
Btw we get the way to live migrate such instances as well and currently re-deploying computes in the regions to support that - fra1 is ready, kna1 in progress, sto2 planned.

Reply:
Being able to live-migrate an instance with block storage migration doesn't change the fact that the application must take care of its own HA and data availability, because if the VM uses local storage only it's not protected against catastrophic loss of a single compute node.

Reply:
You just mentioned in previous section {{migrate within the Cloud infrastructure, without interruption}} which I read as refference to live migration and not to shared storage. As if we're talking there about failover during hypervisor failure there will be interruptions.
But yes, you're totally right about this section - if there was no intention to talk about ability to live migrate instances it should be left as is.